x-pack/plugins/integration_assistant/common/api/model/common_attributes.gen.ts

```diff
@@ -85,6 +97,14 @@ export const SamplesFormat = z.object({
    * For some formats, specifies whether the samples can be multiline.
    */
   multiline: z.boolean().optional(),
+  /**
+   * For CSV format, specifies whether the samples have a header row.
+   */
+  header: z.boolean().optional(),
+  /**
+   * For CSV format, specifies the columns guessed at by the LLM.
```

Comment:
Sure LLM guesses them , but for API Spec documentation `generated by the LLM` probably sounds good.

Reply:
I am fine with that.